app/assets/html/sidebar.html

```diff
@@ -4,10 +4,10 @@
             <div ui-view="sidebar-matching" class="sidebar-matching-container"></div>
 
             <li ng-repeat="gladiatorId in gladiatorIds">
-                <div class="gladiator-heading-container menu-item" ng-click="toggleGladiatorPanel(gladiatorId)">
+                <div class="gladiator-heading-container menu-item" ng-click="goToProfile(gladiatorId)">
```

Comment:
Some design considerations here...does it make more sense to link to the profile only if you click the profile picture or username, instead of the entire container element? It seems to me that gladiators would want to expand the view to see their conversations more often than they would want to see others' profiles.